app/model/jobs/ReindexSteps.scala

```diff
@@ -1,44 +1,64 @@
 package model.jobs
 
 import model.{Section, Tag}
-import play.api.libs.functional.syntax._
-import play.api.libs.json._
+import org.cvogt.play.json.Jsonx
 import repositories._
+import play.api.Logger
+import play.api.libs.json._
 import services.{Config, KinesisStreams}
+import scala.util.control.NonFatal
 
-// Tags
-
-case class ReindexTags(capiJobId: String) extends Step {
+case class ReindexTags() extends Step {
   override def process: Option[Step] = {
-    TagLookupCache.allTags.get.grouped(Config().reindexTagsBatchSize).foreach { tags =>
-      KinesisStreams.reindexTagsStream.publishUpdate(capiJobId, Tag.createReindexBatch(tags))
+    val total = TagLookupCache.allTags.get.size
```

Comment:
I know what you mean, and it's worth the debate, but I think for this it should be using the Dynamo directly even if we keep caches.